Core/src/org/sleuthkit/autopsy/report/ReportHTMLConfigurationPanel.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2018 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.report;
+
+import org.sleuthkit.autopsy.coreutils.ModuleSettings;
+
+/**
+ * The panel shown for all TableReportModules when configuring report modules.
+ */
+@SuppressWarnings("PMD.SingularField") // UI widgets cause lots of false positives
+public class ReportHTMLConfigurationPanel extends javax.swing.JPanel {
```

Comment:
This class and its methods can be package scope I believe as they do not appear to be accessed outside of the report package.